src/pages/MemberProfilePage.tsx

```diff
@@ -34,10 +35,12 @@ import {
   FaChartBar,
   FaUser,
 } from 'react-icons/fa';
+import { getToday } from '../localization';
 
 const selectLastHalfYear = (contributions: Activity[]) => {
-  const currentYear = new Date().getFullYear();
-  const currentMonth = new Date().getMonth();
+  const today = getToday();
+  const currentYear = today.getFullYear();
+  const currentMonth = today.getMonth();
```

Comment:
I don't think this changes anything from the previous way, if that's the same result you want then this is fine. You can use getUtcMonth() if there's a bug with showing stuff.

Reply:
yeah I don't think it's any different, will change to UTC month/year